### DISCUSSION TITLE
Problem with sentryone plan explorer extension

### DISCUSSION THREAD

**Discussion by hajerty** [microsoft/azuredatastudio#20634] [Q&A]
Hi since the last 2 months the subject extension doesn't work because I receive this error "The .NET Core Runtime is required (version 2.1+), but cannot be found. Ensure it is installed, and in your path." when ADS starts.
I downloaded more and more version of .net core to fix the problem but without effects on the error. I suppose that ADS cannot find the .net core path of installed version.

Anyone has the same error and how I can fix it? Thanks in advance

**Comment by Charles-Gagnon** [comment]
You should contact Sentry One about this - we don't own or maintain that extension in this repo 

https://docs.sentryone.com/help/plan-explorer-azure-data-studio-extension

(there's a support link on the sidebar you could try)
